### PR TITLE
Gpr 2023.01.28

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

**Review by Avuxo — APPROVED** [review]
otsukare

**Review by yk1512 — APPROVED** [review]
🙏

File: script/Ciel/Day 10/10_01_CIEL10_4_2C.txt [hunk 1]
```diff
@@ -160,7 +160,7 @@ I have to say, though―――
 [sha:295b8d92ac1b14f2e98430a45ada9bed298c05a3]{
 -- Page 228, Offset 27690.
 -- 可愛いだけじゃなく、凜々しくて、かっこよくて、シエル先輩に似合ってました」
-They aren't just cute――they're cool, dignified, and suit you to a tee."
+That outfit isn't just cute――it's cool, dignified, and suits you to a tee."
 }
 [sha:f6377d4ffe0b86e8d01b5c9606a608f0f28f7044]{
 -- Page 229, Offset 27691.
```
Comment (Petrikow) [inline script/Ciel/Day 10/10_01_CIEL10_4_2C.txt:166]: This line needs to be changed to reflect the above.

>Well, it's basically just a uniform...

File: script/Ciel/Day 10/10_01_CIEL10_5.txt [hunk 2]
```diff
@@ -572,7 +572,7 @@ that I'm alive, and the wonders life entails, as something to be celebrated.
 -- Page 320, Offset 27938.
 -- ……うん、そうでした。ごめんなさい、バカなコトを言ってしまって。わたしが憧れたのは、貴方のそういう強さでした」
 Yes... I should have known.
- I'm sorry for saying something stupid.
+ I'm sorry for saying something silly.
  That strength is exactly what I admired about you in the first place."
```
Comment (Petrikow) [inline script/Ciel/Day 10/10_01_CIEL10_5.txt:576]: "so silly" feels like it works better here.

File: script/QA/QA_15.txt [hunk 3]
```diff
@@ -16,7 +16,7 @@ Seems like all you Executors have a couple screws loose.%{n}
 [sha:0e6b7fc851cc7ef4b3d1f543adf297b87788ce02]{
 -- Page 56, Offset 43422.
 -- なに？　ストレスたまるの、シエルん<職|と><場|こ>？
- What, are your workplaces really that stressful?
+What, are your workplaces really that stressful?
```
Comment (Petrikow) [inline script/QA/QA_15.txt:19]: Plural feels a lil weird with this phrasing, or is that just me?
>...is your workplace...

**Pull request merged**